Sources/Logging/Logging.swift

```diff
@@ -39,20 +39,41 @@ import WASILibc
 /// logger.info("Hello World!")
 /// ```
 public struct Logger {
-    @usableFromInline
-    var handler: LogHandler
+    /// A private property to hold the boxed `LogHandler`.
```

Comment:
as a unit test, you can put `XCTAssertEqual(MemoryLayout<UnsafeRawPointer>.size, MemoryLayout<Logger>.size)`. This should pass once this is complete (and it won't pass today).